crates/core/src/wrpc.rs

```diff
@@ -132,7 +132,7 @@ impl Client {
     ///
     /// ## Arguments
     /// * `nats` - The NATS client to use for communication.
-    /// * `lattice` - The lattice to use for communication.
+    /// * `prefix` - The lattice and component ID to use for communication, e.g. "default.echo".
```

Comment:
can we make these separate args? let's not force people to concat strings before they call this.

Reply:
Yes I like this idea, since that's the way we're going to use it always